### ISSUE TITLE
New nightly breaks xla.device()

### ISSUE THREAD

**Issue by abhishekkrthakur** [pytorch/xla#1927]
it seems that new nightly has broken xla.

to reproduce in colab:

```
!curl https://raw.githubusercontent.com/pytorch/xla/master/contrib/scripts/env-setup.py -o pytorch-xla-env-setup.py
!python pytorch-xla-env-setup.py --version nightly --apt-packages libomp5 libopenblas-dev
```

then: `import torch_xla.core.xla_model as xm`

now when I do: `xm.xla_device()`, its stuck forever.

TPU is available though.

**Comment by dlibenzi** [comment]
We have found the issue and we are opened a P0 issue internally.

**Comment by jysohn23** [comment]
Thanks for reporting the issue @abhishekkrthakur! I'm talking to other engineers internally to get it fixed.

To unblock yourself and other Kaggle users please use this setup instead (`nightly` -> `20200415`):
```
!curl https://raw.githubusercontent.com/pytorch/xla/master/contrib/scripts/env-setup.py -o pytorch-xla-env-setup.py
!python pytorch-xla-env-setup.py --version 20200415 --apt-packages libomp5 libopenblas-dev
```

**Comment by jysohn23** [comment]
@abhishekkrthakur I've gone ahead and pinned `nightly` to yesterday's version so even if you update to nightly again just now, it should work again.

**Comment by tmabraham** [comment]
@jysohn23 I think there's a potentially related issue going on. Since yesterday, [this Kaggle Kernel](https://www.kaggle.com/tanlikesmath/xlm-roberta-pytorch-xla-tpu) will not run on nightly but will run on version `20200420`. Do you know if this is related?

**Comment by jysohn23** [comment]
Hey @tmabraham, can you post the full error stack?

**Comment by JackCaoG** [comment]
@tmabraham full error stack will be super helpful. We know that `torch.rand` and `torch.randn` is broken on the nightly, working on a fix now.

**Comment by tmabraham** [comment]
@jysohn23 There is no error. Upon further investigation it seems that this kernel gets stuck in the `train_loop_fn` when printing the loss. In that kernel, I have `mesh_reduce` to reduce the loss from all the cores, but it turns out if I just print with `master_print` or regular `print`, it still doesn't work. 

Maybe there is a bug in my code, but it's weird that everything was working before yesterday, and now my code breaks and gets stuck.

**Comment by tmabraham** [comment]
@jysohn23 Just wanted to follow up on this. I just tested my code again and I am still having this issue. It's not urgent (I can just use the older version) but just thought I'd let you know.

**Comment by jysohn23** [comment]
Thanks for the update! I'll keep it in my things to do and circle back when I have time in that case. Thanks.

**Comment by apthagowda97** [comment]
I am getting the same error for **nightly** as well as **20200415** in Kaggle

`  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  3727  100  3727    0     0  39231      0 --:--:-- --:--:-- --:--:-- 39231
Updating TPU and VM. This may take around 2 minutes.
Updating TPU runtime to pytorch-dev20200415 ...
Found existing installation: torch 1.4.0
Uninstalling torch-1.4.0:
Done updating TPU runtime: <Response [200]>
  Successfully uninstalled torch-1.4.0
Found existing installation: torchvision 0.5.0
Uninstalling torchvision-0.5.0:
  Successfully uninstalled torchvision-0.5.0
Copying gs://tpu-pytorch/wheels/torch-nightly+20200415-cp36-cp36m-linux_x86_64.whl...
- [1 files][ 86.9 MiB/ 86.9 MiB]                                                
Operation completed over 1 objects/86.9 MiB.                                     
Copying gs://tpu-pytorch/wheels/torch_xla-nightly+20200415-cp36-cp36m-linux_x86_64.whl...
\ [1 files][115.4 MiB/115.4 MiB]                                                
Operation completed over 1 objects/115.4 MiB.                                    
Copying gs://tpu-pytorch/wheels/torchvision-nightly+20200415-cp36-cp36m-linux_x86_64.whl...
/ [1 files][  2.5 MiB/  2.5 MiB]                                                
Operation completed over 1 objects/2.5 MiB.                                      
ERROR: torch-nightly+20200415-cp36-cp36m-linux_x86_64.whl is not a supported wheel on this platform.
ERROR: torch_xla-nightly+20200415-cp36-cp36m-linux_x86_64.whl is not a supported wheel on this platform.
ERROR: torchvision-nightly+20200415-cp36-cp36m-linux_x86_64.whl is not a supported wheel on this platform.`

**Comment by jysohn23** [comment]
Hi, thanks for reporting the issue. It looks like Kaggle updated their python version without us knowing from 3.6 to 3.7. We'll talk to them and see what we can do. Thanks.

cc: @ifigotin

**Comment by ifigotin** [comment]
We'll temporarily rollback to 3.6 on TPU VMs.

**Comment by ifigotin** [comment]
The default should already be 3.6 for most by now.

**Comment by apthagowda97** [comment]
Yes it's working now, Thanks

**Comment by abhishekkrthakur** [comment]
Thank you everyone.
